deploy/kubernetes/helm/sloth/templates/_helpers.tpl

```diff
@@ -39,3 +39,12 @@ app.kubernetes.io/instance: {{ .Release.Name }}
 {{- define "sloth.chart" -}}
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
 {{- end }}
+{{- define "sloth.imagePullSecrets" -}}
+{{- range .Values.imagePullSecrets }}
+  {{- if eq (typeOf .) "map[string]interface {}" }}
```

Comment:
This is confusing, why allow multiple types if what we require is a list of secrets?

Reply:
I’m assuming to allow for both the case where individuals provide a list of mappings (- name: my-secret) and the case where they provide a simple list of raw secret names (- my-secret). 

The flexibility is fine but I also don’t mind a strict values interface here. Id lean more towards the list of mappings being the preferred approach but that’s just my opinion.

Reply:
yes exaclty, wanted to cover both situations

Reply:
I would prefer only to allow one way of doing things, in the long term it's easier for users and maintainers.

Reply:
i understand, i added just for list version

Reply:
https://github.com/slok/sloth/pull/400/commits/93ec2c4d578c93cd4c286387e1776900ea1efd23

Reply:
Awesome! many thanks @muhammetozekli!